### ISSUE TITLE
new commands - SMO Vulnerability and Assessment

### ISSUE THREAD

**Issue by potatoqualitee** [sqlcollaborative/dbatools#6366]
Export-SqlVulnerabilityAssessmentBaselineSet
Export-SqlVulnerabilityAssessmentScan
Get-SqlAssessmentItem
Get-SqlSensitivityClassification
Import-SqlVulnerabilityAssessmentBaselineSet
Invoke-SqlAssessment
Invoke-SqlVulnerabilityAssessmentScan
New-SqlVulnerabilityAssessmentBaseline
New-SqlVulnerabilityAssessmentBaselineSet
Remove-SqlSensitivityClassification
Set-SqlSensitivityClassification

**Comment by pelsenator** [comment]
Hi there

I don't see the relevant "Remove-DbaLinkedServer" for "Copy-DbaLinkedServer". I'm using the dbatools.io solution to keep instance level objects in-sync on AOAG clusters. Due to GDPR regulation the sys admin access for server agent accounts had to be removed across the board and I had to resort to the use of PS credentials instead (What a delight). Without the Remove-DbaLinkedServer command I will have to make use of native SQL statements which means I have to resort to impersonation rights. This comes with it's own set of headaches, something I tried to avoid in the first place.

Regards
Pelser
P.S. - Thanks a million for all the hard work and effort with dbatools.io. Best set of DBA tools ever!
